content/zh/examples/admin/resource/quota-mem-cpu-pod-2.yaml

```diff
@@ -9,8 +9,8 @@ spec:
     resources:
       limits:
         memory: "1Gi"
-        cpu: "800m"
+        cpu: "800m"      
       requests:
         memory: "700Mi"
         cpu: "400m"
-
+      
```

Comment:
Extra spaces added. pls remove.

Reply:
The original English yaml has extra spaces, so these two files are not updated.
I detected it by using a script
done